src/main/java/org/terasology/projsndwv/components/MatedComponent.java

```diff
@@ -18,14 +18,42 @@
 import org.terasology.entitySystem.Component;
 import org.terasology.entitySystem.entity.EntityManager;
 import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.logic.inventory.ItemDifferentiating;
 import org.terasology.projsndwv.genetics.components.GeneticsComponent;
 
-public final class MatedComponent implements Component {
+/**
+ * Stores the genetic and lifespan information for a queen bee.
+ */
+public final class MatedComponent implements Component, ItemDifferentiating {
+    /**
+     * An entity containing the genetics component of the drone that mated with the queen.
+     */
     public EntityRef container;
 
+    /**
+     * The number of ticks remaining in the lifespan of the queen.
+     */
+    public int ticksRemaining;
+
+    /**
+     * The total ticks in the entire lifespan of this queen.
+     */
+    public int lifespan;
+
+    @SuppressWarnings("unused")
```

Comment:
At present, the convention is a comment in the javadoc over a suppress warnings.